config/database.yml

```diff
@@ -0,0 +1,9 @@
+---
+# This should be filled out and placed in /chap-press/config/database.yml
+# This file is *only* used to generate wp-config.php
+staging:
+  host: 'localhost' 
+  database: 'chappress_dev' 
+  username: 'chappress'
+  password: 'chappress'
```

Comment:
If these need to be updated after being copied to server, values should be changed to more clearly signal that:

```
staging:
  host: 'TBA'
  database: 'chappress_staging'
  username: 'chappress'
  password: 'TBA'
```